arkane/encorr/ae.py

```diff
@@ -134,6 +144,49 @@ def execute(self, output_file: str = None):
                 logging.warning('Could not write atom energies to database. Captured error:')
                 logging.warning(str(e))
 
+    def save_correlation_mat(self, path: str, labels: List[str] = None):
+        """
+        Save a visual representation of the parameter correlation matrix.
+
+        Args:
+            path: Path to save figure to.
+            labels: Parameter labels.
+        """
+        try:
+            import matplotlib.pyplot as plt
```

Comment:
Unbury this in both AEC and BAC

Reply:
I look into this part of code again, and my new thought is that plotting functions are not necessary for the Arkane job execution to fail, so it makes sense to do this

Reply:
RE in-person discussion: make the import look like this: https://github.com/ReactionMechanismGenerator/RMG-Py/blob/main/rmgpy/ml/estimator.py#L121